File: Easy/1D Bush Fire/1D Bush Fire.js
const N = parseInt(readline());
for (let i = 0; i < N; i++) 
{
    let a = 0;
    const line = readline();
    for (let j = 0; j < line.length; j++) 
    {
        if (line[j] == 'f')
        {
            a = a + 1;
            j = j + 2;
        }
    }
    console.log(a);
}

File: Easy/1D Bush Fire/1D Bush Fire.vb
Module Solution

    Sub Main ()
        
        Dim N as Integer
        N = Console.ReadLine()

        For i as Integer = 0 To N-1
            Dim a as Integer
            a = 0
            Dim line as String
            line = Console.ReadLine()
            For c As Integer = 0 To line.Length()-1
                If line(c)="f" Then
                    a = a+1
                    c = c+2
                End If
            Next
            Console.WriteLine(a)
        Next
    End Sub
End Module